src/test_solvers/main.cpp

```diff
@@ -0,0 +1,32 @@
+#include "solvers.h"
+int main(){
+
+    // Initialize triplets for the matrix
+    std::vector<int> rowInd;
```

Comment:
If you want, you can initialize these using `{ ...}` initializers. I think it might make it more readable

```c++
std::vector<int> rowInd = { 0, 1, 1};
std::vector<int> colInd = {1, 0, 1};
std::vector<float> val = {1, 2, 3};
```